docs/3-reference/useCheckoutPricingReturn.stories.mdx

```diff
@@ -0,0 +1,63 @@
+import { Subtitle } from '@storybook/addon-docs/blocks';
+import { PropsTable } from '@storybook/components';
+
+<Meta title="Reference/UseCheckoutPricingReturn" />
+
+# UseCheckoutPricingReturn
+
+<Subtitle slot={() => 'useCheckoutPricing() return value'} />
+
+<PropsTable
+  rows={[
+    {
+      name: 'UseCheckoutPricingReturn',
+      type: { summary: 'array', detail: "[{ useCheckoutPricingState }, setCheckoutPricing]" },
+      description:
+        'An array with a `useCheckoutPricingState` object as the first item and a `setCheckoutPricing` update function as the second value',
+    },
+  ]}
+/>
+
+
+### UseCheckoutPricingState
+
+<PropsTable
+  rows={[
+    {
+      name: 'price',
+      type: { summary: 'object' },
+      description:
+        'A recurly.js [checkoutPricing pricing.price object](https://developers.recurly.com/reference/recurly-js/#pricing)',
+    },
+    {
+      name: 'pricing',
+      type: { summary: 'object' },
+      description:
+        'A recurly.js [checkoutPricing pricing object](https://developers.recurly.com/reference/recurly-js/#pricing)',
+    },
+    {
+      name: 'loading',
+      type: { summary: 'boolean' },
+      description: 'Describes the current loading state of useCheckoutPricing',
+    },
+  ]}
+/>
+
+### SetCheckoutPricing
```

Comment:
ed: capitalization is incorrect.

Reply:
I capitalized all the headers for the types. Would camelcasing everything be more appropriate?